.cci.jenkinsfile

```diff
@@ -33,10 +33,24 @@ cosaPod {
     unstash name: 'build'
     fcosBuild(skipKola: true, overlays: ["install"])
 
+    // This is a temporary hack we use during spec stabilization to work around
+    // external tests which use experimental versions. It should be commented
+    // out in normal times.
+    if (shwrapRc("grep -nr 3.3.0-experimental /srv/fcos/src/config/tests/kola") != 0) {
+        // Ideally here, we would simply sed the tests, like this:
+        //shwrap("find /srv/fcos/src/config/tests/kola -type f -exec sed -i 's/3.3.0-experimental/3.3.0/' {} \\;")
+
+        // But it doesn't work because kola still wants to parse the config. We
+        // should teach it to not do that. For now, just nuke the ones we know
```

Comment:
> But it doesn't work because kola still wants to parse the config. We should teach it to not do that.

Filed https://github.com/coreos/coreos-assembler/issues/2249 to discuss this.